webpack.test.config.js

```diff
@@ -1,65 +1,12 @@
-var webpack = require( 'webpack' ),
-	path = require( 'path' ),
-	nodeExternals = require( 'webpack-node-externals' );
+const webpack = require( 'webpack' );
+const path = require( 'path' );
+const nodeExternals = require( 'webpack-node-externals' );
+const webpackBaseConfig = require( './webpack.config' );
 
-module.exports = {
```

Comment:
Those were the kinds of differences that pushed towards this "merger" in the first place. It was difficult to keep the two in sync and the differences were minor and annoying to manage.

Technically we don't need the sass to compile for tests but the time spent on it (anecdotally at least) is not significant enough to worry about. 

The main thing is that the tests continue to run as expected and aren't slow, which they aren't.